app/controllers/resources_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ResourcesController < ApplicationController
+  def index
+    category_id = params[:category_id]
+    @resources = if category_id
+                   Category.find(category_id).resources
```

Comment:
An alternative:

 `Resource.where category_id: category_id`

There are semantic and performance differences between the two. `Category.find ...` will result in a 404 if the category doesn't exist, whereas `Resource.where ...` will simply return no results. `Category.find ...` performs two BD queries, one for the category and a second for the resources; whereas `Resource.where ...` performs one.